### DISCUSSION TITLE
PixArt local install help

### DISCUSSION THREAD

**Discussion by Woisek** [PixArt-alpha/PixArt-alpha#69] [Q&A]
Could someone maybe provide a step-by-step tutorial on how to install everything needed on WIndows, to get PixArt running on a local install with the Gradio UI? Or make even an installer for it? That would be awesome.

Thanks in advance.

**Comment by lawrence-cj** [comment]
Here is one: https://youtu.be/ZiUXf_idIR4

**Reply by Woisek** [reply]
Thanks 👍, but I prefer free resources and not be nudged into a pay service, just for installing an open source software. 🙂

**Reply by lawrence-cj** [reply]
Well. If you use ComfyUI. Here is one way: https://github.com/city96/ComfyUI_ExtraModels

**Reply by Woisek** [reply]
That surely looks interesting, and I will give it a try, thanks for this link. 👍
But still I would love to have something like this -> https://huggingface.co/spaces/PixArt-alpha/PixArt-alpha installed locally. 🙂
